PresentScreenings/ScreeningsPlanner.cs

```diff
@@ -93,32 +122,37 @@ private void DisplayResultsOfRating(FilmRating rating, string filmFan, List<Film
             // Display summary for this rating.
             int highRatedFilmCount = films.Count;
             int plannedFilmCount = films.Where(f => HasAttendedScreening(f, filmFan)).Count();
-            string fmt = "{0} out of {1} films with rating {2} planned for {3}.";
-            _builder.AppendFormat(fmt, plannedFilmCount, highRatedFilmCount, rating, filmFan);
+            string str = $"{_logTime()}  {plannedFilmCount} out of {highRatedFilmCount} films with rating {rating} planned for {filmFan}.";
+            _builder.AppendFormat(str);
 
             // Display films with this rating that weren't planned.
             if (highRatedFilmCount - plannedFilmCount > 0)
             {
-                string unplannedFmt = "Films with rating {0} that could not be planned for {1}:";
-                _builder.AppendLine();
-                _builder.AppendFormat(unplannedFmt, rating, filmFan);
+                string unplannedStr = $"Films with rating {rating} that could not be planned for {filmFan}:";
                 _builder.AppendLine();
+                _builder.AppendLine(unplannedStr);
```

Comment:
Please consider removing variable `unplannedStr` and use the assigned string explicitly.

Reply:
Done.
Plus done in the other case. That line was already very long anyway.